datafusion/common/src/config.rs

```diff
@@ -311,6 +311,10 @@ config_namespace! {
 
         /// Should DataFusion keep the columns used for partition_by in the output RecordBatches
         pub keep_partition_by_columns: bool, default = false
+
+        /// If true, the parquet reader will replace `Utf8/Utf8Large` with `Utf8View`,
```

Comment:
I think this config option only applies to Parquet at the moment, rather than applying in a general to execution

What do you think about putting them on the ParquetOptions struct instead? 

https://github.com/apache/datafusion/blob/4dd8532e6cd52c480a29a7851c6676a69f261545/datafusion/common/src/config.rs#L323

Reply:
It applies to other formats as well, as the transformation is done after we call:
```rust
let mut schema = self.format.infer_schema(state, &store, &files).await?;

... // transformation
```

It should work for any format.

Reply:
Cool, so let's just update the comments 

```suggestion
        /// If true, listing tables will read columns of `Utf8/Utf8Large` with `Utf8View`,
```